main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # Open and load JSON file to "players" variable
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+        for player in players:
+            race = players[player]["race"]
+            skills = players[player]["race"]["skills"]
+            guild = players[player]["guild"]
+            # Create Race objects if records not exist yet
+            if not Race.objects.filter(name=race["name"]).exists():
+                Race.objects.create(
+                    name=race["name"],
+                    description=race["description"]
+                )
+            # Create Guild objects if records not exist yet
+            if guild:
```

Comment:
Can you explain for what you check?

Reply:
If the character in the json file has a guild equal to null, then at the stage of creating an entry in the table I will get an error, here I filter out null values.